src/gm3/util.js

```diff
@@ -436,3 +436,23 @@ export function metersAreaToUnits(meters, units) {
             return meters;
     }
 }
+/** Converts numeric lengths between given units
+ *
+ * @param {number} length - Length
+ * @param {string} srcUnits - Source unit
+ * @param {string} destUnits - Destination unit
+ * @return {number} Converted length
+ */
+export function convertLength(length, srcUnits, destUnits) {
+    // US survey feet, miles
+    const equivalentMeters = {
```

Comment:
As a small optimization (and to make the constants available to others, this should get moved OUT of the function and put into it's own declaration.  E.G. "const EQUIVALENT_METERS = { ..}" on line 439.